index.js

```diff
@@ -394,6 +408,8 @@ class YouTubePlayer extends EventEmitter {
       this.load(this.videoId)
     }
 
+    this.emit('ready')
```

Comment:
I think this is the wrong place to emit this event. If the user runs code in the 'ready' listener then it will run before the queued events which are run by `_flushQueue`.